lib/src/types/webhook.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Type of events that can be subscribed
+ */
+export type WebhookEvent = 'item/created' | 'item/updated' | 'item/error' | 'all'
```

Comment:
Improvement suggestion: 

We could provide all of these union types like the following instead:

```tsx
export const WEBHOOK_EVENTS = ['item/created', 'item/updated', 'item/error', 'all'] as const
export type CountryCode = typeof COUNTRY_CODES[number]
/**
 * @typedef WebhookEvent 
 * Type of events that can be subscribed
 */
export type WebhookEvent = typeof WEBHOOK_EVENTS[number]
```
The result for `WebhookEvent` type is equivalent as before, but with the advantage of providing the actual list of possible values, which is useful for clients for further validations / type inferencing, type guards and more. Otherwise they would have to manually build this list themselves which can get outdated on some change. 

This applies to all union types exported: `ConnectorType`, `ConnectorCredential.type`, `AccountType`, etc..

Reply:
Updated.